apollo-normalized-cache/src/main/java/com/apollographql/apollo/cache/normalized/lru/EvictionPolicy.kt

```diff
@@ -0,0 +1,96 @@
+package com.apollographql.apollo.cache.normalized.lru
+
+import com.apollographql.apollo.api.internal.Optional
+import java.util.concurrent.TimeUnit
+
+/**
+ * Controls how long a [com.apollographql.apollo.cache.normalized.Record] will
+ * stay in a [LruNormalizedCache].
+ */
+class EvictionPolicy internal constructor(
+    private val maxSizeBytes: Optional<Long>,
```

Comment:
These are also private. Let's remove `Optional`s here too. You can take a look at this class for reference. Let's make these public and expose as `val maxSizeBytes: Long?`

Then we can keep the below functions with `Optional` for backward compatibility reasons and deprecate them. 

https://github.com/apollographql/apollo-android/blob/master/apollo-normalized-cache-api/src/main/java/com/apollographql/apollo/cache/normalized/NormalizedCache.kt#L129-L130

Reply:
Once I make the changes, should I updated the invocations in `LruNormalizedCache`?

Reply:
Especially if it already converted to Kotlin, why not.